startup/routes.js

```diff
@@ -17,6 +18,7 @@ module.exports = function (app) {
 
   app.use('/api', limit)
   app.use('/api/v1/users', example)
+  app.use('/api/reservations', reservations)
```

Comment:
you can add v1 as you want 
api/v1/...

it says what api version we use. I guess we dont create next version so its optional - as you want